pkg/cmd/cli/backup/create.go

```diff
@@ -290,7 +290,8 @@ func (o *CreateOptions) BuildBackup(namespace string) (*velerov1api.Backup, erro
 			return nil, err
 		}
 		if o.Name == "" {
-			o.Name = schedule.TimestampedName(time.Now())
+			o.Name = schedule.TimestampedName(time.Now().UTC())
+			//o.Name = schedule.TimestampedName(time.Now())
```

Comment:
Could you please review this line?

Reply:
Thanks. This line should be removed, but I'm not sure why the documents update is also included in the updated PR.
It seems it's better to open a new PR.

Reply:
Yes, please check why the documents update is included in this PR.

Reply:
OK. I will close this PR, and continue on the new one.